src/main/java/pl/betoncraft/betonquest/StaticEvents.java

```diff
@@ -121,7 +125,7 @@ private long getTimestamp(final String hour) {
          * @param timeStamp
          * @param eventID
          */
-        public EventTimer(final long timeStamp, final EventID eventID) {
+        public EventTimer(final long timeStamp, final EventID[] eventID) {
```

Comment:
How about using `...` instead of `[]`? Though isn't really important as it isn't a public API.
```suggestion
        public EventTimer(final long timeStamp, final EventID... eventID) {
```